client/src/components/dashboard/Files.js

```diff
@@ -39,40 +43,34 @@ function Files() {
   useEffect(() => {
     setFeedbacks(allFeedbacks);
   }, [allFeedbacks]);
+
   return (
     <>
       <section>
-        <section>
-          {feedbacks.length !== 0 &&
-            feedbacks[0].feedback_id !== null &&
-            feedbacks.map((feedback) => (
-              <div>
-                <div key={feedback.feedback_id}>{feedback.feedback_text}</div>
-                <button onClick={() => deleteFeedback(feedback.feedback_id)}>
-                  delete feedback
-                </button>
-              </div>
-            ))}
-        </section>
         <div className="icon-heading">
-          <h1>Files</h1>
+          <h2>Files</h2>
           <img alt="files icon" src={filesIcon} />
         </div>
 
         <div className="login-signin-buttons">
           <Link to="/new-feedback">
```

Comment:
(An interactive element cannot contain another interactive element. Don't wrap buttons with links. )

Buttons trigger actions or processes. Links trigger navigation only. Is this a link or a button? You may wish to add a class to apply the button styles.